src/scss/1_base/tables.scss

```diff
@@ -14,19 +14,80 @@
 //
 // Style guide: base.table
 
+$table-border-color: #666 !default;
+
 table {
   // Add vertical rhythm margins.
   @include leader(1);
   @include trailer(1);
   // Remove most spacing between table cells.
   border-collapse: collapse;
   border-spacing: 0;
-  // Prevent cramped-looking tables
+  // Prevent cramped-looking tables.
   width: 100%;
+
+  &.table-fixed-width {
+    table-layout: fixed;
+  }
+
+  p {
+    margin: .9rem 0;
+
+    &:first-child {
+      margin-top: 0;
+    }
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
 }
 
 td,
 th {
+  @include adjust-font-size-to(14px, 3.5);
```

Comment:
Could replace this with @include responsive-typo-map($small-text); and remove further font-size adjustments line 58 and 63